lib/utils/interfaces/dispatch.interface.ts

```
@@ -0,0 +1,111 @@
+  phone: string;
+  available: boolean;
+  zone: {
+    _id: string;
+    title: string;
+  };
+}
+
+export interface IGetRidersByZone {
+  ridersByZone: IRidersByZone[];
+}
+export interface IGetRidersByZoneVariables {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)